src/index/vocabulary/PolymorphicVocabulary.h

```diff
@@ -159,6 +159,21 @@ class PolymorphicVocabulary {
         vocab_);
   };
 
+  // Check if `getGeoInfo` can return results because there is a
+  // `GeoVocabulary`.
+  bool isGeoInfoAvailable() const {
+    return std::visit(
+        [](const auto& vocab) {
+          using T = std::decay_t<decltype(vocab)>;
+          if constexpr (ad_utility::isInstantiation<T, SplitVocabulary>) {
+            return vocab.isGeoInfoAvailable();
+          } else {
+            return false;
+          }
+        },
```

Comment:
That is again too brittle for my taste, 
please implement it similar to the other functions s.t. a static assert fails when we add another vocab for which we don't specify whether it might contain geoInfos.